### ISSUE TITLE
Think about what to leave out

### ISSUE THREAD

**Issue by flying-sheep** [da-x/prustle-rs#1]
Since e.g. `os.path` and `shutils` are superseeded by `pathlib` in python.

And `pathlib` is basically already part of rust via `std::path`

**Comment by da-x** [comment]
Good point, though tons of Python scripts still rely on `os.path` and friends, and the older APIs persist in the mind of many programmers. I'd say since it's sort of of a `compat` package, we can support both options.

**Comment by flying-sheep** [comment]
sure, but maybe a bit of guidance towards greener pastures is useful
